model/ftn/ww3_bounc.ftn

```diff
@@ -66,6 +69,9 @@
 !     - Can be used also to diagnose contents of nest.ww3 file
 !       in read mode
 !
+!     - Input spectra are assumed to be formulated on a standard
+!       pole. However, the model grid can be on a rotated pole.
+!
```

Comment:
This is an important note. A user might do mistakes if not aware of this. May be mentioned in the Manual Sect. 3.4.9 Rotated grids in a separate, short paragraph like: "If the user wants to apply boundary condition from spectral files using one of the programs ww3_bounc or ww3_bound as described in Appendix B.2, the input spectra are assumed to be formulated on a standard pole."

Reply:
Thanks for the review @CarstenHansen . I will make a small addition the to manual to ensure the user is aware that input spectra should be on a standard pole.

Reply:
I have also added some comments in the relevant `.inp `and `.nml `files.

Reply:
If you are happy with the manual/documentation changes, @CarstenHansen , then I will merge and close this PR. Thanks.